### PR TITLE
chore(localdebug): add local certificate installation driver

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Add `tools/install` driver to trust development certificate.
```yaml
provision:
  - name: prerequisite installation
    uses: tools/install
    with:
      devCert:
        trust: true
```

### CODE REVIEW

File: packages/fx-core/src/component/driver/tools/installDriver.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { StepDriver } from "../interface/stepDriver";
```
Comment (swatDong) [inline packages/fx-core/src/component/driver/tools/installDriver.ts:4]: code style - better order

File: packages/fx-core/src/component/driver/tools/installDriver.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+const ACTION_NAME = "tools/install";
+const outputName = Object.freeze({
+  SSL_CRT_FILE: "SSL_CRT_FILE",
+  SSL_KEY_FILE: "SSL_KEY_FILE",
+});
+const helpLink = "https://aka.ms/teamsfx-actions/tools/install";
+
+@Service(ACTION_NAME)
+export class ToolsInstallDriver implements StepDriver {
+  @hooks([addStartAndEndTelemetry(ACTION_NAME, TelemetryConstant.PROVISION_COMPONENT_NAME)])
```
Comment (swatDong) [inline packages/fx-core/src/component/driver/tools/installDriver.ts:25]: why provision component?

Comment (XiaofuHuang) [reply packages/fx-core/src/component/driver/tools/installDriver.ts]: Sorry, moved it to deploy life cycle.

File: packages/fx-core/src/component/driver/tools/installDriver.ts (hunk 1 above)
Comment (XiaofuHuang) [reply packages/fx-core/src/component/driver/tools/installDriver.ts]: Thx!

**Review by swatDong — APPROVED** [review]

File: packages/fx-core/src/component/driver/tools/installDriver.ts (hunk 2 above)
Comment (swatDong) [reply packages/fx-core/src/component/driver/tools/installDriver.ts]: Just curious - it seems some drivers use actionName as component, while others use provision/deploy... Any rule?

**Review by 1yefuwang1 — DISMISSED** [review]

File: packages/fx-core/src/component/driver/tools/installDriver.ts (hunk 2 above)
Comment (XiaofuHuang) [reply packages/fx-core/src/component/driver/tools/installDriver.ts]: > Just curious - it seems some drivers use actionName as component, while others use provision/deploy... Any rule?

As @blackchoey discussed offline, there will be another story to unify the driver telemetry. Before telemetry format is decided, we can use `ACTION_NAME` now.

**Review by swatDong — APPROVED** [review]

**Review by kuojianlu — DISMISSED** [review]

**Review by 1yefuwang1 — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**